games/gamesModel.js

```diff
@@ -0,0 +1,24 @@
+let gamesList = [];
+
+function addGame(game) {
+  gamesList.push(game);
+  return gamesList.find(g => g.name === game.name);
+}
+
+function doesGameExist(game) {
+  console.log(game);
+  const myGame = gamesList.filter(g => g.name === game.name);
+  if (myGame.length > 0) {
+    return true;
+  } else {
+    false;
```

Comment:
There's no return here! (so this returns `undefined` in the false case, which is still falsey, but not `false`).

Also, more succinct (and still readable) would be:
```javascript
// ...
const myGame = gamesList.filter(g => g.name === game.name);
// return the boolean resulting from the condition itself
return myGame.length > 0;
```